src/objects/zcl_abapgit_object_intf.clas.abap

```diff
@@ -528,6 +528,7 @@ CLASS zcl_abapgit_object_intf IMPLEMENTATION.
       " HERE: switch with feature flag between XML and JSON file format
       IF mv_aff_enabled = abap_true.
         ls_intf = read_json( ).
+        lcl_aff_metadata_handler=>deserialize_translation( mo_files ).
```

Comment:
Help wanted: With this method, I try to actually write the translation content to `SEOCOMPOTX` and a-likes.
However, after deserialization, the seocompotx is not filled with translation.
@mbtools could it be, that the deserialization at a later stage overrides/allocates-initial the seocompotx entries? I'm not super familiar with the step, so this is just a guess.

Reply:
Your method needs to fill the corresponding structures in ls_intf

Reply:
Thanks, will try.